src/opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -776,10 +776,10 @@ RawKeyword * newRawKeyword( const std::string& deck_name, ParserState& parserSta
 }
 
 // This code is made to skip part of the eclipse file containing the contents of UDT
-// UDT is currently not a asupported keyword. 
+// UDT is currently not a asupported keyword.
```

Comment:
since you touch the line maybe you can fix the typo while at it?

Reply:
sorry, was my editor that removed the trainling spaces. Just realized. I'm planning to remove all the trailing spaces to not touch all these files, but I can fix this typo if you want me to.

Reply:
right. in that case let's just deal with it separate from this PR to keep the changes minimal.

Reply:
https://github.com/OPM/opm-common/pull/2601 takes care of this